### DISCUSSION TITLE
Manually set scene id for network objects / Prefabs

### DISCUSSION THREAD

**Discussion by Hassan-Nasser** [FirstGearGames/FishNet#504] [Feature Request]
Request to add manual method of setting scene id for network objects and prefabs
This will improve the work flow of separate projects for client/server structured projects

**Comment by brinca** [comment]
I've posted a similar request (for the same reason), but it's a bit more descriptive: #704

**Comment by FirstGearGames** [comment]
Closing due to other posts.
